tests/TestCase/ORM/Rule/LinkConstraintTest.php

```diff
@@ -202,7 +202,7 @@ public function testInvalidRepository($options): void
             ]],
         );
         $Articles->shouldReceive('buildRules')
-            ->atLeast()->once()
+            ->between(1, 2)
```

Comment:
Using `->atLeast()->once()` means "from once to infinite times". Better to use the `between()` method when it is a known range.